about.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>AGRI EXPO</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <section id="aboutpage">
+        <header>
+          <div id="hamburger">
+                <span></span>
+                <span></span>
+                <span></span>
+          </div>
+          <div id="nav1">
+            <a href="#"><img src="./Images/facebook.svg"></a>
+            <a href="#"><img src="./Images/twitter.svg"></a>
+            <a href="#">English</a>
+            <a href="#">My Page</a>
+            <a href="#">Log-in</a>
+          </div>
```

Comment:
> Kindly wrap this section in a nav tag so as to follow HTM semantic tags usage. 😊

- [ ] I would advise you to wrap this section in a nav tag as recommended by the reviewer so that it would be accessible by other devices such as screenreaders and tab.